tests/tpp.sh

```diff
@@ -3,3 +3,7 @@
 # run again to check for idempotency
 ./tpp/manage.sh
 
+# Some assertions
+set -e
+grep -q SimonDavy@OPENCORONA ~bloodearnest/.ssh/authorized_keys
```

Comment:
what an old-school assertions framework!

Reply:
I've got a plan for a better one, but it'll have to wait